src/sharechain/in_memory.rs

```diff
@@ -652,6 +637,24 @@ impl ShareChain for InMemoryShareChain {
             .build()?)
     }
 
+    async fn get_tip_and_uncle_blocks(&self) -> Result<Vec<Arc<P2Block>>, ShareChainError> {
```

Comment:
This was previously unused. Is it used now?

Reply:
yeah